src/main/java/se/iths/SpringbootGroupProject/entities/User.java

```diff
@@ -0,0 +1,75 @@
+package se.iths.SpringbootGroupProject.entities;
+
+import jakarta.persistence.*;
+import org.hibernate.proxy.HibernateProxy;
+
+import java.util.Objects;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true)
+    private String userName;
+    @Lob
+    private Byte[] image;
+    private String name;
```

Comment:
```suggestion
    private String firstName;
    private String lastName;
```

or a method to combine the name

setName(String first, String last)
name = first + " " + last;